src/coreclr/vm/threads.cpp

```diff
@@ -1136,6 +1136,11 @@ extern "C" void *JIT_WriteBarrier_Table_Loc;
 void *JIT_WriteBarrier_Table_Loc = 0;
 #endif // TARGET_ARM64
 
+#ifdef TARGET_LOONGARCH64
+extern "C" void *JIT_WriteBarrier_Loc;
+void *JIT_WriteBarrier_Loc = 0;
+#endif // TARGET_LOONGARCH64
+
 #ifdef TARGET_ARM
 extern "C" void *JIT_WriteBarrier_Loc = 0;
 #endif // TARGET_ARM
```

Comment:
nit: these if-def blocks (line 1123 to 1146) can be reduced to just:
```c
#ifdef TARGET_X86
extern "C" void *JIT_WriteBarrierEAX_Loc;
#else
extern "C" void *JIT_WriteBarrier_Loc;
void *JIT_WriteBarrier_Loc = 0;
#endif

#ifdef TARGET_ARM64
extern "C" void (*JIT_WriteBarrier_Table)();
extern "C" void *JIT_WriteBarrier_Table_Loc;
void *JIT_WriteBarrier_Table_Loc = 0;
#endif // TARGET_ARM64
```

Reply:
OK, Thanks